bppy/model/event_selection/simple_event_selection_strategy.py

```diff
@@ -37,10 +37,13 @@ def selectable_events(self, statements):
                     possible_events = {x for x in possible_events if x not in statement.get('block')}
         return possible_events
```

Comment:
```suggestion
        if len(possible_events) > 0:
            return possible_events
        else:
            return None
```

Reply:
https://github.com/bThink-BGU/BPpy/issues/4

Reply:
**this check is done in the select method already:**
```python
    def select(self, statements, external_events_queue=[]):
        selectable_events = self.selectable_events(statements)
        if selectable_events:
            return random.choice(tuple(selectable_events))
        else:
            if len(external_events_queue) > 0:
                return external_events_queue.pop(0)
            else:
                return None
```